packages/renderer/src/lib/dashboard/ReleaseNotesBox.svelte

```diff
@@ -0,0 +1,79 @@
+<script lang="ts">
+import { faCircleArrowUp } from '@fortawesome/free-solid-svg-icons';
+import { Button, CloseButton, Link } from '@podman-desktop/ui-svelte';
+import { onMount } from 'svelte';
+
+import Markdown from '../markdown/Markdown.svelte';
+
+let showBanner = true;
+let updateAvilable = false;
+let currentVersion: string;
+let title: string = '';
+let summary: string = '';
+let imageUrl: string = '';
+let imageAlt: string = '';
+
+async function getInfoFromNotes() {
+  let curVersionSplit = currentVersion.split('.', 2);
```

Comment:
@deboer-tim add after line 16 `currentVersion = '1.12.0'`. I've had this line before for testing purposes, but the test failed with it